setup.py

```diff
@@ -2,9 +2,16 @@
 
 setup(
     name="city_metrix",
-    version="0.1.0",
+    version="0.1.1",
     description="Module to calculate various metrics on cities.",
-    packages=["city_metrix", "city_metrix.layers", "city_metrix.metrics"],
+    packages=["city_metrix", "city_metrix.layers", "city_metrix.metrics", "city_metrix.models", "city_metrix.models.building_classifier"],
+    include_package_data=True,
+    package_data={
+        'city_metrix.models.building_classifier': [
+            'building_classifier.pkl',
+            'V2-building-class-data.geojson',
```

Comment:
@chrowe was wondering if the user needs this file for running the package? Also, could you rename is to something more like `building-class-data-v2.geojson`?

Reply:
The user needs `building_classifier.pkl` to generate the `SmartSurfaceLULC` layer, but the geojson is not necessary. The geojson is the training data for `building_classifier.pkl` that I got from Elizabeth, I removed it for now.